crossmodal/features_extractor/audio_features/pitch.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np 
+from librosa.core import piptrack
+
+def pitch(sound = None,
+          fmin = 80,
+          fmax = 8000,
+          sr = None,
+          window_length = 20,
+          overlap = 5,
+          threshold = 0.1):
+    
+    '''
+        In this function we use the librosa.core.piptrack function in order to locate the pitches of the give utterance. We know that piptrack uses the QIFFT and so we give the appropriate parameters in order to calculate the spectrogram and the pitch array.
+        I will not write the parameters since it is quite obvious what this method takes as input.
+    '''
+
+    # Check if the sound and the sampling rate were given.
+    assert sound is not None, 'Error, no sound np.ndarray was given.'
```

Comment:
do not use assert as part of the logic. Just use an if and raise an appropriate exception.